src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASubmissionStatus.tsx

```diff
@@ -227,9 +228,11 @@ const useGetButtonURL = (
 
   if (!submission) return null
 
-  const previousStep = submissionID === submission.externalID && step
+  // Do not link to the initial artist step.
+  const previousStep =
+    submissionID === submission.externalID && step !== INITIAL_STEP && step
```

Comment:
To prevent linking to the Artist step in case it's the saved previous submission.
<img width="755" alt="Screenshot 2024-08-12 at 11 40 19" src="https://github.com/user-attachments/assets/d42d5d97-6f1f-402a-9c41-8d3c1acba823">